sdk/core/core-websockets/test/utils/vitest.ts

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { chaiAzure } from "@azure-tools/test-utils-vitest";
+import { chai, should as shouldFn, assert, expect } from "vitest";
+import chaiAsPromised from "chai-as-promised";
+
+chai.use(chaiAsPromised);
+chai.use(chaiAzure);
+const should = shouldFn();
+export { should, expect, assert };
```

Comment:
What do we get here from chai that vitest itself can't do from the `expect` API?

Reply:
Good point, removed.